src/zope/publisher/http.py

```diff
@@ -547,7 +554,8 @@ def _authUserPW(self):
             encoded = self._auth.split(None, 1)[-1]
             decoded = base64.b64decode(encoded.encode('iso-8859-1'))
             name, password = bytes.split(decoded, b':', 1)
-            #name, password = base64.b64decode(encoded.encode('ascii')).split(':', 1)
+            # name, password = base64.b64decode(
+            #     encoded.encode('ascii')).split(':', 1)
```

Comment:
Why not remove it entirely? Commented out code should not exist.

Reply:
okay.